src/D3Hierarchy.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState, useRef } from "react";
+import * as d3 from 'd3'
+
+import repairsdbData from '../assets/repairs_db';
+
+const D3Hierarchy = () => {
+    const ref = useRef();
+
+    useEffect(() => {
+        if (repairsdbData) {
+
+            const format = d3.format(",");
+            const nodeSize = 17;
+            const root = d3.hierarchy(data).eachBefore((i => d => d.index = i++)(0));
```

Comment:
```suggestion
            const root = d3.hierarchy(repairsdbData).eachBefore((i => d => d.index = i++)(0));
```

This fails because you haven't renamed this variable

Reply:
ah ok, changed it
wish I had a local dev environment (weeps)